scripts/generate_mrenclave.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if [ "$1" == "" ]; then
+    echo 'Parse the output of the SGX tool to extract the MRENCLAVE value'
+    echo 'and save it to file $TCF_HOME/metadata_info.txt.'
+    echo 'Usage: generate_mrenclave.sh <library location>'
+    exit
+fi
+
+OUT_FILE="${TCF_HOME}/metadata_info.txt"
```

Comment:
Please add an option to pass a optional command line argument for OUT_FILE location. If it's not provided the script can do a default location.

Reply:
Done